plugins/node/opentelemetry-instrumentation-mysql/src/instrumentation.ts

```diff
@@ -278,10 +287,20 @@ export class MySQLInstrumentation extends InstrumentationBase<
           values = [_valuesOrCallback];
         }
 
-        span.setAttribute(
-          SemanticAttributes.DB_STATEMENT,
-          getDbStatement(query, format, values)
-        );
+        let dbStatement = '';
+        if (typeof thisPlugin._config.dbStatementSerializer === 'function') {
+          dbStatement = getDbStatement(query, format, values);
+          dbStatement = thisPlugin._config.dbStatementSerializer(dbStatement);
+        } else {
+          if (typeof query === 'string') {
+            dbStatement = getDbStatement(query, format, values);
+            dbStatement = defaultDbStatementSerializer(dbStatement);
+          } else {
+            dbStatement = query.sql;
+          }
```

Comment:
This code makes it so if I invoke the driver with an object
```js
connection.query({ sql: 'SELECT * FROM "customers"'});
```

It will record `db.statement: 'SELECT * FROM "customers"'`

But the same query with a string will produce a different result:
```js
connection.query('SELECT * FROM "customers"');
```
`db.statement: 'SELECT * FROM ?'`

It will also behave differently when the parameters are embedded into the query string and not supplied as values:
```
connection.query({ sql: 'SELECT * FROM customers WHERE password="1234"'}); // db.statement: SELECT * FROM customers WHERE password="1234"

connection.query('SELECT * FROM customers WHERE password="1234"'); // db.statement: SELECT * FROM customers WHERE password=?

```

I think it is preferable to have consistency with how the instrumentation operates by default.